.github/DISCUSSION_TEMPLATE/rfc.yml

```
@@ -0,0 +1,33 @@
+    id: alternatives
+    attributes:
+      label: Alternatives Considered
+  - type: textarea
+    id: impact
+    attributes:
+      label: Risks / Impact
+  - type: textarea
+    id: rollout
+    attributes:
+      label: Rollout Plan
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)